SparkPost/2web2ui
# FE-1425 - fix empty state segment call going out

https://sparkpost.atlassian.net/browse/FE-1425

### What Changed

- Added a pre pending boolean to prevent setting empty state before the data call has been made.

### How To Test

- Load up this branch and stage side by side. 
- Visit the domains list pages while looking at segment. 
- Hard refresh the page to see if the `Empty State Loaded` event goes through.
- The `Empty State Loaded` event should not show up when there are domains returned from the api call. 

### To Do
